index.html

```diff
@@ -0,0 +1,247 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Youtube Page</title>
+    <!-- firest -->
+    <link rel="shortcut icon" href="./img/download.png" type="image/png">
+    <!-- font awasome icons -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css" integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="./css/style.css">
+    <script src="./js/script.js" defer></script>
+</head>
+<body>
+
+    <nav>
+        <div class="container">
+            <div class="humbager">
+                <i class="fa-solid fa-bars" i="humbager-icon" id="sidebar"></i>
+                <div class="logo">
+                    <i class="fa-brands fa-youtube"></i>
+                    <h1>YouTube</h1>
+                </div>
+
+            </div>
+            <div class="search-nav">
+                <div class="search">
+                    <input type="search" placeholder="   Search">
+                    <div class="icon-div">
+                        <i class="fa-solid fa-magnifying-glass"></i>
+                    </div>
+                </div>
+                <div class="icon-microphone">
+                    <i id="microphone" class="fa-solid fa-microphone"></i>
+
+                </div>
+
+            </div>
+            <div class="profile">
+                <i class="fa-solid fa-video-slash"></i>
+                <i class="fa-regular fa-bell"></i>
+                <div class="profile-image">
+                    <img src="./img/profile-img.avif" alt="">
+                </div>
+
+            </div>
+
+        </div>
+    </nav>
+
+    <main>
+        <div class="container">
+            <aside>
+                <ul class="nav-menu">
+                    <li class="nav-item">
+                        <i class="fa-solid fa-house"></i>
+                        <a href="#" class="nav-link">Home</a>
+                    </li>
+                    <li class="nav-item">
+                        <i class="fa-solid fa-bolt"></i>
+                        <a href="#" class="nav-link">Shorts</a>
+                    </li>
+                    <li class="nav-item">
+                        <i class="fa-brands fa-alipay"></i>
+                        <a href="#" class="nav-link">Subscription</a>
+                    </li>
+                    <li class="nav-item">
+                        <i class="fa-brands fa-playstation"></i>
+                        <a href="#" class="nav-link">Library</a>
+                    </li>
+                    <li class="nav-item">
+                        <i class="fa-solid fa-download"></i>
+                        <a href="#" class="nav-link">Downloads</a>
+                    </li>
+                </ul>
+
+            </aside>
+            <div class="video">
```

Comment:
- [Optional] Great job for adding the comment politely and with reasons 👍,  I think it would be better if you use a couple of emojis and/or gifs in the review, they make the code review more human-friendly 👽🤖.